src/main/java/edu/umich/its/cpm/EmailFormatter.java

```diff
@@ -74,6 +140,53 @@ public String getBody() {
         return prunedHeaders;
     }
 
+    public ArrayList<String> mboxFormatHeaders() {
+        ArrayList<String> headers = prunedHeadersWithoutContentType();
+        String startingHeader = mboxStartingHeader(headers);
+        headers.add(0, startingHeader);
+        return headers;
+    }
+
+    public String mboxStartingHeader(ArrayList<String> headers) {
+        String from = null;
+        String date = null;
+        StringBuilder startingHeader = new StringBuilder();
+        for (String header : headers) {
+            // From: Jane Doe <jdoe@umich.edu>
```

Comment:
Not needed, I just did it for readability sake and it is one liner so no harm done